nltk/parse/transitionparser.py

```diff
@@ -490,7 +490,7 @@ def _create_training_examples_arc_eager(self, depgraphs, input_file):
         print(" Number of valid (projective) examples : " + str(countProj))
         return training_seq
 
-    def train(self, depgraphs, modelfile):
+    def train(self, depgraphs, modelfile, verbose=True):
```

Comment:
@stevenbird 
`TransitionParser` used `verbose=True` as default value to train the SVM model (see [line 525][1] before my PR). Conversely, `sklearn.svm.SVC` uses `verbose=False` as default (see [sklearn.svm.SVC][2]).

I made this variable dynamic so that it can be changed by users (and it's useful to clean tests output), but I did not change the behaviour that was originally implemented by @longdt219. If you think this has to be fixed, I can work on it.

[1]: https://github.com/nltk/nltk/pull/1501/files#diff-ac01d9460dcfd9e576978fbf2885bb87L525
[2]: http://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html